src/Command/Config/ExportSingleCommand.php

```diff
@@ -197,9 +202,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $module = $input->getOption('module');
         $configName = $input->getArgument('config-name');
         $optionalConfig = $input->getOption('optional-config');
+        $uuid = $input->getOption('uuid');
 
-        $config = $this->getConfiguration($configName);
-
+        if ($uuid) {
+            $config = $this->getConfiguration($configName, TRUE);
```

Comment:
DrupalConsole follows PSR-2 code style.
* `TRUE` must be `true`
* `FALSE` must be `false`